jerry-main/main-unix-snapshot.c

```diff
@@ -391,13 +391,15 @@ process_generate (cli_state_t *cli_state_p, /**< cli state */
 
   if (literals_file_name_p != NULL && !is_import_literals)
   {
-    const size_t literal_buffer_size = jerry_parse_and_save_literals ((jerry_char_t *) source_p,
-                                                                      source_length,
-                                                                      false,
-                                                                      output_buffer,
-                                                                      sizeof (output_buffer) / sizeof (uint32_t),
-                                                                      is_save_literals_mode_in_c_format);
-    if (literal_buffer_size == 0)
+    uint32_t *lit_out_buf_p = output_buffer + snapshot_size;
+    size_t lit_out_buf_sz = (sizeof (output_buffer) / sizeof (uint32_t)) - snapshot_size;
```

Comment:
This expression also seems to be incorrect. `sizeof (output_buffer)` is already in bytes, isn't it?

Reply:
Extra note / question: What ensures that the output of `jerry_get_literals_from_snapshot` will fit in the buffer of size `lit_out_buf_sz`? And how could we detect if it didn't?